sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -170,9 +218,33 @@ export interface RecognizeLinkedEntitiesSuccessResult extends TextAnalyticsSucce
     readonly entities: LinkedEntity[];
 }
 
+// @public (undocumented)
+export interface SentenceAspect {
```

Comment:
Some of the types in the API file here are dangling (not used by any other result type):

- `SentenceAspect`
- `SentenceOpinion`
- `SentenceOpinionSentiment`
- `SentenceAspectSentiment`
- `AspectRelation`
- `AspectRelationType`
- `AspectConfidenceScoreLabel`

These are probably getting exported from the generated layer, but since we are remapping the values in them to custom convenience layer types, these aren't needed anymore, so we can safely remove them.

Reply:
done.